src/components/navigators/BottomNavigator/BottomNavigator.tsx

```diff
@@ -0,0 +1,58 @@
+"use client";
+
+import Link from "next/link";
+import { usePathname } from "next/navigation";
+
+import { Home, HomeFill, Exploration, ExplorationFill, Diary, DiaryFill } from "@/components/icons";
+import * as S from "./BottomNavigator.style";
+import { BOTTOM_NAV_INABLE } from "../navInablePath";
+
+interface BottomNavItemProps {
+  path: string;
+  icon?: JSX.Element;
+  activeIcon?: JSX.Element;
+  label: string;
+  isProfile?: boolean;
+}
+
+const BottomNavigator = () => {
+  const currentPathName = usePathname();
+
+  const navAble = !BOTTOM_NAV_INABLE.includes(currentPathName);
+
+  const isPathActive = (path: string) => currentPathName === path;
+
+  const bottomNavItems: BottomNavItemProps[] = [
+    { path: "/home", icon: <Home />, activeIcon: <HomeFill />, label: "홈" },
+    { path: "/explore", icon: <Exploration />, activeIcon: <ExplorationFill />, label: "탐색" },
+    { path: "/user/id/diary", icon: <Diary />, activeIcon: <DiaryFill />, label: "내 기록" },
+    { path: "/user/id", label: "마이페이지", isProfile: true },
+  ];
+
+  if (!navAble) return null;
+
+  return (
+    <S.BottomNavContainer>
+      {bottomNavItems.map(({ path, icon, activeIcon, label, isProfile }) => (
+        <Link
+          href={path}
+          key={path}
+          title={label}
+        >
+          <S.BottomNavItem active={isPathActive(path)}>
+            {isProfile ? (
+              <S.ProfileCircle active={isPathActive(path)} />
+            ) : isPathActive(path) ? (
+              activeIcon
+            ) : (
+              icon
+            )}
```

Comment:
[ P3 ] 개인적으로는 삼항 연산자가 2번 연속으로 쓰이면 읽기가 많이 들더라구요.. 🤦
그래서 저는 보통 이런경우 요렇게 사용하는편이라 한번 의견 드려봅니다
 
```suggestion
{ isProfile && <S.ProfileCircle active={isPathActive(path)} /> }

{ !isProfile && isPathActive(path) ? activeIcon : icon }
```

Reply:
엇 이거 제가 알기론 린트에서도 잡아주는 걸로 알고 있는데..! 송희님 혹시 따로 Caution은 발생하지 않으셨나요? 👀